tests/templates/test_transforms.py

```diff
@@ -0,0 +1,42 @@
+import numpy as np
+from pennylane.templates.transforms import adjoint
+import pennylane as qml
+
+def test_adjoint_sanity_check():
+
+	dev = qml.device("default.qubit", wires=1)
+
+	def my_op():
+		qml.RX(0.123, wires=0)
+		qml.RY(2.32, wires=0)
+		qml.RZ(1.95, wires=0)

```

Comment:
Indentation in this file looks a bit off 🤔

Reply:
Gah, tabs vs spaces. (black doesn't run on test code)